tests/util.go

```diff
@@ -123,12 +146,18 @@ func WaitUntilResourceReadyByLabelTestNamespace(resourceType, label, query, expe
 
 func CreateUser(username string) {
 	By(fmt.Sprintf("Wait until user %s is created ", username))
-	execute(Result{cmd: "oc", verb: "create", resourceType: "user", resourceName: username})
+	_, stderr, err := ktests.RunCommandWithNS("", "oc", "create", "user", username)
+	if !strings.Contains(stderr, fmt.Sprintf("Error from server (AlreadyExists): users.user.openshift.io \"%s\" already exists", username)) {
+		ktests.PanicOnError(err)
+	}
 }
 
 func DeleteUser(username string) {
```

Comment:
This function should return an `error`. Otherwise, there is no way how to analyze whether it was executed successfully.

Reply:
I have updated with the changes